modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -23,7 +23,26 @@ To understand the parsing strategies that DataWeave readers and writers can appl
 [[format_xml_cdata]]
 == CData Custom Type
 
-`CData` is a custom DataWeave data type for XML that is used to identify a Character DATA (CData) block. The `CData` type tells the XML writer to wrap content inside a `CData` block or to check for an input string inside a `CData` block. In DataWeave, `CData` inherits from the type `String`.
+`CData` is a custom DataWeave data type for XML that is used to identify a character data (CDATA) block. The `CData` type makes the XML writer wrap content inside a `CDATA` block or to check for an input string inside a `CDATA` block. In DataWeave, `CData` inherits from the type `String`.
+
+== Doctype Custom Type
```

Comment:
```suggestion
== DocType Custom Type
```

Reply:
Fixing in data-weave master https://github.com/mulesoft/data-weave/pull/2042/files 
need to cherry pick to support branch as well

Reply:
fixed via PR above